README.md

```diff
@@ -118,8 +118,8 @@ Run `terraform destroy` when you don't need these resources.
 - [cloudwatch-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/cloudwatch-logs) - Send logs from `CloudWatch`.
 - [s3](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/s3) - Send logs from `S3` bucket.
 - [eventbridge](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/eventbridge) - Send logs from `eventbrdge`.
-- [firehose](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/firehose) - Send logs from `firehose`.
-- [vpc-flow-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/vpc-flow-logs) - Send `vpc flow logs` logs from `S3` bucket.
+- [firehose](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/firehose) - Send metrics from `firehose`.
+- [vpc-flow-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/vpc-flow-logs%20) - Send `vpc flow logs` logs from `S3` bucket.
```

Comment:
If we would like to correct this the naming for metrics should be cloudWatch-metric-stream for metrics, and each scenario we are supporting for logs